CONTRIBUTING.md

```diff
@@ -425,11 +465,7 @@ related issues in our [Contributors chat room](https://gitter.im/FreeCodeCamp/Co
 
 #### Important: ALWAYS EDIT ON A BRANCH
 
-Take away only one thing from this document: Never, **EVER**
-make edits to the `staging` branch. ALWAYS make a new branch BEFORE you edit
-files. This is critical, because if your PR is not accepted, your copy of
-staging will be forever sullied and the only way to fix it is to delete your
-fork and re-fork.
+Take away only one thing from this document: Never, **EVER** make edits to the `staging` branch. ALWAYS make a new branch BEFORE you edit files. This is critical, because if your PR is not accepted, your copy of staging will be forever sullied and the only way to fix it is to delete your fork and re-fork.
```

Comment:
> the only way to fix it is to delete your fork and re-fork

Not true, you can `git reset --hard upstream/staging`, you will lose any commits outside of the upstream commit history, but there is no need to re-fork.

Reply:
Totally true. This is some formatting WIP. I am going to remove all of this outdated git steps. These can be made lot simpler.